scratch/main_test.go

```diff
@@ -68,7 +68,8 @@ func BenchmarkWASMCallOverhead(b *testing.B) {
 
 		mod, err := r.InstantiateModuleFromBinary(ctx, wasmBytes)
 		if err != nil {
-			log.Panicln(err)
+			slog.Error(err.Error())
+			return
```

Comment:
should panic or fail the test no?

Reply:
Omg, I'm sorry for this. It was a mistake. I changed that. 

One question though. Why don't we use `b.Error(err)` instead of `panic(err)`?

Reply:
I probably just did not know about it, `b.Error` is fine if it works.

Actually I think we want `b.Fatal`: https://pkg.go.dev/testing#B.Fatal

Reply:
Good catch; I will replace all `panic()` with `b.Fatal` or `t.Fatal` then